src/Database/Driver/Sqlserver.php

```diff
@@ -64,8 +64,14 @@ public function connect()
             return true;
         }
         $config = $this->_config;
+
+        // PDO::ATTR_PERSISTENT is not allowed by SQL Server PHP drivers.
+        // See: https://github.com/Microsoft/msphpsql/issues/65
+        if (isset($config['persistent']) && $config['persistent']) {
+            throw new \InvalidArgumentException('Sqlserver driver does not allow config setting: persistent');
```

Comment:
What about? ``Enabling persistent connections in Sqlserver by setting the config key "persistent" to "true" is not supported, as the SQL Server PHP driver does not support persistent connections``.

My goal is to make the error messag as clear as possible, as discussed in https://github.com/cakephp/cakephp/issues/10752